dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/dispatcher/internal/channel/HttpDispatcherLink.java

```diff
@@ -1260,4 +1260,14 @@ public boolean useForwardedHeaders() {
         return false;
     }
 
+    /**
+     * Calls function to set the supress 0 byte chunk flag.
+     */
+    public void setSuppressZeroByteChunk(boolean suppress0ByteChunk) {
+        if (this.isc != null) {
```

Comment:
Might be useful to log if this method isn't able to set the flag due to the isc being null. Nothing code breaking, just QoL change for easier debugging.